op-service/flags/flags.go

```diff
@@ -0,0 +1,77 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/ethereum-optimism/optimism/op-node/chaincfg"
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+)
+
+const (
+	RollupConfigFlagName   = "rollup.config"
+	NetworkFlagName        = "network"
+	CanyonOverrideFlagName = "override.canyon"
+	DeltaOverrideFlagName  = "override.delta"
+)
+
+func CLIFlags(envPrefix string) []cli.Flag {
+	return []cli.Flag{
+		&cli.Uint64Flag{
+			Name:    CanyonOverrideFlagName,
+			Usage:   "Manually specify the Canyon fork timestamp, overriding the bundled setting",
+			EnvVars: opservice.PrefixEnvVar(envPrefix, "OVERRIDE_CANYON"),
+			Hidden:  false,
+		},
+		&cli.Uint64Flag{
+			Name:    "override.delta",
```

Comment:
```suggestion
			Name:    "DeltaOverrideFlagName",
```

Though why do we need shared canyon/delta override flags here?

Reply:
This is to keep RollupConfig consistent across services (in case we need canyon / delta time somewhere else, op-program, etc)